infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -196,6 +196,35 @@
     "id": 34,
     "reason": "The selected Region is not valid.",
     "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ProvisioningFailureException",
+    "id": 35,
+    "reason": "An error occurred handling your request. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.UnspecifiedProvisioningFailureException",
+    "id": 36,
+    "reason": "An unspecified error occurred handling your request. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ProvisioningTimeOutException",
+    "id": 37,
+    "reason": "Your request was not complete within the configured time-out. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ProvisioningMaxRetriesExceededException",
+    "id": 38,
+    "reason": "Your request failed after the maximum number of configured re-try attempts. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ProvisioningReplicaFailureException",
+    "id": 39,
+    "reason": "A ReplicaSet failed to be provisioned for your request. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
```

Comment:
@manstis Could you add a task to the backlog for us to review the contents of `exceptionInfo.json`. It's grown organically and I think we need to put in place better definitions on when to add things to it.

cheers,

Rob